366: Spalte Haus-Nr. ergänzen bei Export Gebäude

joergreichert opened the pull request:
Task-Url: http://polymap.org/kaps/ticket/366

Hausnummer und Hausnummer-Zusatz in Gebäudeexport

joergreichert committed:
366: Spalte Haus-Nr. ergänzen bei Export Gebäude 

Task-Url: http://polymap.org/kaps/ticket/366

Hausnummer und Hausnummer-Zusatz in Gebäudeexport

stundzig commented:
Änder das mal, das bei einer fehlenden Hausnummer eben "" ins Resultat geschrieben wird und nicht abgebrochen. Hausnummer kann schonmal fehlen.

joergreichert committed:
366: Spalte Haus-Nr. ergänzen bei Export Gebäude 

Task-Url: http://polymap.org/kaps/ticket/366

Kein Abbruch, wenn Hausnummer fehlt
